flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/resources/KubernetesOwnerReference.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.kubeclient.resources;
+
+import io.fabric8.kubernetes.api.model.OwnerReference;
+import io.fabric8.kubernetes.api.model.OwnerReferenceBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+/**
+ * Represent Toleration resource in kubernetes.
+ */
+public class KubernetesOwnerReference extends KubernetesResource<OwnerReference> {
+
+	static final String API_VERSION = "apiversion";
```

Comment:
I had them private initially, but decided to make them public, so that they can be used for programmatic definitions.
When I am writing code that creates owner reference, its handy to have them available

Reply:
IIUC, you mean to refer these variables in your own java program, not Flink source codes. Since `KubernetesOwnerReference` is an internal class, which should not be directly used out of Flink. So I am afraid it does not make sense to set these variables `public`.